web/client/codechecker_client/cmd_line_client.py

```diff
@@ -49,6 +49,30 @@ def init_logger(level, stream=None, logger_name='system'):
     LOG = logger.get_logger(logger_name)
 
 
+def report_to_report_data(report, context=None):
```

Comment:
Wouldn't be better to do add this type conversion method to the `Report` class?

Reply:
`Report` class can be found in [`report.py`](https://github.com/Ericsson/codechecker/blob/master/codechecker_common/report.py) which can be found in the `codechecker_common` package. As we discussed personally, we would like to eliminate the `codechecker_common` package. As far as this function is used only by this module I think it's better to leave it in this file.

Reply:
Ok, keep it here for now, we will see later how the refactoring changes the report.py in the common module.